packages/cli-upload/src/utils.js

```diff
@@ -20,7 +20,7 @@ export async function getImageResources({
   absolutePath
 }) {
   let rootUrl = `http://localhost/${encodeURIComponent(name)}`;
-  let imageUrl = `http://localhost/${encodeURIComponent(relativePath)}`;
+  let imageUrl = `http://local/${encodeURIComponent(relativePath)}`;
```

Comment:
1. lets make it consistent throughout -  say `render.percy.local` etc
2. we need to test this properly, we don't want any regression for existing users.

Reply:
Lets keep it local but consistent across